imports/ui/layouts/app-body.js

```diff
@@ -69,7 +68,7 @@ Template.App_body.helpers({
     ] });
   },
   activeListClass(list) {
-    const active = ActiveRoute.name('Lists.show')
+    const active = FlowRouter.current().route.name === 'Lists.show'
```

Comment:
```js
import { FlowRouter , RouterHelpers } from 'meteor/ostrio:flow-router-extra';

RouterHelpers.name('Lists.show');
```

See [docs for more details](https://github.com/VeliovGroup/flow-router/blob/master/docs/helpers/RouterHelpers.md)